src/yew/mod.rs

```diff
@@ -60,14 +59,12 @@ impl Component for CodeEditor {
         }
 
         Self {
-            props: ctx.props().clone(),
             node_ref: NodeRef::default(),
             editor: None,
         }
     }
 
-    fn changed(&mut self, ctx: &Context<Self>) -> bool {
-        let old_props = mem::replace(&mut self.props, ctx.props().clone());
```

Comment:
yew 0.20 provide access to `old_props` as a argument to `fn changed` - I presume this works the same, but please correct me if I'm wrong!